.github/workflows/pull_request.yml

```diff
@@ -93,4 +93,4 @@ jobs:
         run: ./mvnw clean install -DskipTests -Pdocker -Dmaven.javadoc.skip=true -T 1C
 
       - name: Run universe tests
-        run: ./mvnw -pl :universe verify -Puniverse -DskipTests -Dmaven.javadoc.skip=true
+        run: ./mvnw -pl :universe verify -Puniverse -Dmaven.javadoc.skip=true
```

Comment:
If you enable unit tests, it will run unit tests along with integration tests, which is not desirable

Reply:
sorry, looks like you are right